bindings/wasm/package.json

```diff
@@ -14,8 +17,14 @@
     "prepublishOnly": "yarn build",
     "test": "mocha"
   },
-  "author": "Lucas Nogueira <lucas.nogueira@iota.org>",
-  "license": "ISC",
+  "contributors": [
+    "Lucas Nogueira <lucas.nogueira@iota.org>"
+  ],
+  "license": "Apache-2.0",
```

Comment:
Why is it not MIT here?

Reply:
My understanding of is that Apache-2.0 is the license for all IOTA projects moving forward, but yes, we should use also use MIT here for historical reasons... (original license is MIT) 

so: MIT or Apache 2 (just like the rust ecosystem) ?

Reply:
yeah, I think it would be great to be MIT or Apache2 like what we usually do on general crates.